Update with recent changes

vsnt opened the pull request:
Update work item progress with community issues #174, #173, #172, #169,

Commit:
Update with recent changes

Update work item progress with community issues #174, #173, #172, #169,

vsnt commented:
Please review @kimdhamilton @wyc. I would like to review with community at tomorrow's mtg.

kimdhamilton commented on index.bs:
```diff
@@ -174,25 +176,45 @@ Group members may propose a different process for their work item, which the cha
 
+The [=task force=] category is not common in the W3C, but has emerged as a useful activity in the CCG to discuss topics of importance to the community.  Task forces must
+- have at least one leader
+- provide a status report each month (meeting minutes will typically suffice)
```
I vote to strike the status report as a "must" requirement for these reasons:
- Introduces possibly unnecessary process. 
- It's a burden on both the chairs to monitor this and the task force to provide status reports.
- Doesn't address what happens if the task forces fail to meet the "must" requirement

The charter and leader don't fall into this category because those only apply on creating, not ongoing requirements.

If we want ongoing requirements, which _may_ make sense, it should be lightweight but also describe what happens if the requirements are not met.

vsnt replied:
@kimdhamilton those were community comments to the Task Force Guidelines doc that no one had any issues with when we discussed at Jan 6 meeting. Where does task force governance go if not in this document?

Task forces have to do minutes as part of being a task force, and the way it is written, the minutes can substitute for them. I had previously had more relaxed timeframe which was overwritten by you and others wanting a specific timeframe (multiple requests for monthly). Task forces must have accountability to the CCG through reporting - whether it is weekly, quarterly or from time to time. 

@wyc will you also weigh in here. Thanks.

wyc replied:
```
The [=task force=] category is not common in the W3C, but has emerged as a useful activity in the CCG to discuss topics of importance to the community. To create a task force, there must be
- at least one leader
- completed requirements for a [=work item=]

For a task force to remain active, it must
- provide a status update report within a timely manner of the chairs' request (meeting minutes will typically suffice)

If a response is not rendered within two weeks of the chairs' request, then the chairs may deem the task force to be inactive.
```

wyc replied:
Would this work for everyone?

vsnt replied:
works for me

kimdhamilton replied:
I think that works. We can get feedback from the community on the response time interval, but I like the idea of spelling out that it's for determining whether the group is active (vs just imposing a process burden for chairs and the task force)

Commit:
Update index.bs

added task for to remain active text, removed requirement for monthly status report (which was per previous community suggestion).
